src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -427,17 +427,17 @@ public void testTooLargeAddEventsSkippedUncompressedRequest() throws Exception {
     assertEquals(0, server.getRequestCount());
 
     addEventsResponse = addEventsClient.log(events).get(5, TimeUnit.SECONDS);
-    assertEquals(addEventsResponse.getStatus(), "success");
-    assertEquals(addEventsResponse.getMessage(), "Skipped due to payload too large");
+    assertEquals("success", addEventsResponse.getStatus());
```

Comment:
Sorry, I should have caught this earlier.  Please change `"success"` to `AddEventsResponse.SUCCESS`.